app/assets/stylesheets/govuk/_errors.scss

```diff
@@ -0,0 +1,61 @@
+/* ERRORS - Form Validation */
+
+/* Why: Match govuk-elements - Change the red */
+
+.flash.error-summary {
+  border: 5px solid $red;
+  margin-top: 30px;
+  padding: 20px 15px 15px
+}
+
+.flash.error-summary a,
+.form-field--error .error-notification{
+  color: $red;
+  font-weight: bold;
+  font-size: 19px;
+}
+
+.form-field--error {
+  border-left: 5px solid $red;
+}
+
+.form-field--error .error-field {
+  border: 4px solid $red;
+}
+
+/* Why: Match govuk-elements - Change spacing */
+
+.form-field--error {
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.error-summary {
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+@media (min-width: 641px) {
+  .error-summary {
+    margin-top: 30px;
+    margin-bottom: 30px;
+  }
+}
+
+/* Why: Match govuk-elements - Change heading size */
+
+.error-summary h2 {
+  font-size: 18px;
+  line-height: 1.2;
+}
+
+error-summary-message {
+
+}
```

Comment:
If error-summary-message is empty, delete?

Reply:
good spot, i'll get it deleted